.eslintrc.js

```diff
@@ -45,9 +48,39 @@ module.exports = {
         node: true
       },
       plugins: ['node'],
-      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
+      extends: ['plugin:node/recommended'],
+      rules: {
         // add your custom rules and overrides for node files here
-      })
-    }
+        "@typescript-eslint/explicit-function-return-type": ["off"],
+        "@typescript-eslint/no-var-requires": ["off"],
+        "@typescript-eslint/camelcase": ["off"]
```

Comment:
I think these can be removed now, `@typescript-eslint` is only used on `*.ts` files and this set of overrides doesn't match any `.ts` files.

Reply:
Ah, thank you!